FrameworkSystem/scripts/dirac-proxy-init.py

```diff
@@ -56,25 +56,25 @@ def getIssuerCert( self ):
     if self.__issuerCert:
       return self.__issuerCert
     proxyChain = X509Chain.X509Chain()
-    result = proxyChain.loadChainFromFile( self.__piParams.certLoc )
-    if not result[ 'OK' ]:
-      gLogger.error( "Could not load the proxy: %s" % result[ 'Message' ] )
+    proxyChainFromFile = proxyChain.loadChainFromFile( self.__piParams.certLoc )
```

Comment:
Why this replacement of 'result' by the name of an "entity" ? I find it much less clear. I try always to indicate that the variable is the result structure. If you want to avoid reusing just 'result' many times, do it resultChainFromFile or resultIssuerCertificate. But keep result as the name prefix.